src/util/apiUtils.js

```diff
@@ -82,14 +82,24 @@ export const parseApiResponse = requestUrl => ([response, body]) => {
 		};
 	}
 	try {
-		value = JSON.parse(body);
-		if (value && value.problem) {
-			value = formatApiError(new Error(`API problem: ${value.problem}: ${value.details}`));
+		if (response.statusCode === 204) { //NoContent response type
+			value = {
+				success: true
+			};
```

Comment:
in fact, thinking about this a bit more, I'm wondering if we should always put the statusCode in the `meta` object for all responses...